src/main/java/frc/robot/constants/RobotConfig.java

```diff
@@ -25,4 +25,12 @@ public static class DriveConfig {
     public static final double MIN_ANGLE_SLEW_RATE = 0.45;
     public static final double MAX_ANGLE_SLEW_RATE = 0.85;
   }
+
+  public static final class IntakeConfig {
+    // TODO: update intake motors default spped
+    public static final double kDefaultSpeed = 0;
```

Comment:
nit: ideally we should state what units this is in

Reply:
Done in commit